config/routes.rb

```diff
@@ -113,7 +113,7 @@
     resource(:miscellaneous_np_info, {only: [:show, :update]})
 
 		namespace(:button) do
-			root({to: :advanced})
+			root({action: :advanced})
```

Comment:
I assume you tested that this page is accessible as before?

Reply:
Yes, it opens the "Advanced" tab when navigating to `/nonprofits/<id>/button`.